antarest/core/config.py

```diff
@@ -654,14 +659,18 @@ class Config:
     tasks: TaskConfig = TaskConfig()
     root_path: str = ""
     api_prefix: str = ""
+    desktop_mode: bool = False
```

Comment:
Note that in order to take this into account in the desktop version we build, you need to add this configuration property in `resources/antares-desktop-fs/config.yaml`.

We also need to update the installer so that it adds that property on existing installations :(
See here: 
https://github.com/AntaresSimulatorTeam/antares-web-installer/blob/main/src/antares_web_installer/config/__init__.py

An alternative would be to not put it in the config file, but to pass this boolean flag when we start the desktop app, in `antarest.desktop.systray_app` (call to `fastapi_app`)